main.tf

```diff
@@ -1,3 +1,28 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+##############################################################################
+# terraform-ibm-scc-workload-protection
+#
+# Creates Security and Compliance Center Workload Protection
+##############################################################################
+
+##############################################################################
+# SCC WP
+##############################################################################
+
+resource "ibm_resource_instance" "scc_wp" {
+  name              = "${var.unique_name}-scc-wp"
```

Comment:
People want full flexibility when naming resources, so please remove the `unique_name` variable, and just call it `name`. Then set this to 
```
name = "${var.name}"
```